hud-data/goog-hud.json

```diff
@@ -4,8 +4,11 @@
   "releaseDate": "2023-09-23",
   "flags": [
     "menus",
+    "minmode",
     "scoreboards",
     "materials"
```

Comment:
Add , at the end here

Reply:
(remind me not to submit these at 3:20 AM where I forget how to read)